src/components/base-address/BaseAddress.vue

```diff
@@ -0,0 +1,539 @@
+                        class="item address-city"
+                        :label="addressCityLabel"
+                        v-model="addressLocal.addressCity"
+                        :rules="[...rules.addressCity, ...spaceRules]"
+          />
+          <v-select v-if="useCountryRegions(addressCountry)"
+                    filled
+                    class="item address-region"
+                    :menu-props="{maxHeight:'40rem'}"
+                    :label="addressRegionLabel"
+                    item-text="name"
+                    item-value="short"
+                    v-model="addressLocal.addressRegion"
+                    :items="getCountryRegions(addressCountry)"
+                    :rules="[...rules.addressRegion, ...spaceRules]"
+          />
+          <v-text-field v-else
+                        filled
+                        class="item address-region"
+                        :label="addressRegionLabel"
+                        v-model="addressLocal.addressRegion"
+                        :rules="[...rules.addressRegion, ...spaceRules]"
+          />
+          <v-text-field filled
+                        class="item postal-code"
+                        :label="postalCodeLabel"
+                        v-model="addressLocal.postalCode"
+                        :rules="[...rules.postalCode, ...spaceRules]"
+          />
+        </div>
+        <div class="form__row">
+          <v-select filled
+                    class="address-country"
+                    :label="addressCountryLabel"
+                    menu-props="auto"
+                    item-text="name"
+                    item-value="code"
+                    v-model="addressLocal.addressCountry"
+                    :items="getCountries()"
+                    :rules="[...rules.addressCountry, ...spaceRules]"
+                    v-on:change="resetRegion()"
+          />
+          <!-- special field to select AddressComplete country, separate from our model field -->
+          <input type="hidden" :id="addressCountryId" :value="addressCountry" />
+        </div>
+        <div class="form__row">
+          <v-textarea auto-grow
+                      filled
+                      class="delivery-instructions"
+                      :label="deliveryInstructionsLabel"
+                      rows="2"
+                      v-model="addressLocal.deliveryInstructions"
+                      :rules="[...rules.deliveryInstructions, ...spaceRules]"
+          />
+        </div>
+      </v-form>
+    </v-expand-transition>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { required } from 'vuelidate/lib/validators'
+import { Component, Mixins, Emit, Prop, Watch } from 'vue-property-decorator'
+import { Validation } from 'vue-plugin-helper-decorator'
+import { uniqueId } from 'lodash'
+
+// NB: local mixin (StoryBook can't find it otherwise)
+import { ValidationMixin, CountriesProvincesMixin } from '@/mixins'
```

Comment:
See if you can import the mixins from the shared package.

Otherwise, the host app will have to provide these mixins in its `@/mixins` folder (possibly via re-import in index file).

Reply:
I tried setting the import to `import { ValidationMixin, CountriesProvincesMixin } from '@bcrs-shared-components/mixins/'` and  whenever I launched the storybook I would get the warning: `Neither '@bcrs-shared-components/mixins/.vue' nor '@bcrs-shared-components/mixins/.js(x)' or '@bcrs-shared-components/mixins//index.js(x)' or '@bcrs-shared-components/mixins//index.ts(x)' could be found in 'C:/Users/TAMUNRO/Dev/Forked/bcrs-shared-components/src/components/base-address'`

Reply:
Does Storybook import that package (in root package.json)?

Reply:
It doesn't. Try it. Or maybe I need to update a config file so SB knows where to find the mixins...

Reply:
Added `    "@bcrs-shared-components/mixins": "^1.1.16",` into root package.json and it seems to have worked. no more warning messages

Reply:
Tay, you resolved this conversation that I started but I didn't have a chance to review it.

What is the status of this issue ("should not be local import of mixins")?

Reply:
Thank you for letting me know... The current import uses `import { ValidationMixin, CountriesProvincesMixin } from '@bcrs-shared-components/mixins'` with no visible issues or warnings

Reply:
Great!